platformio.ini

```diff
@@ -2,8 +2,8 @@
 ;to change the board you want to use uncomment the corect environment in [platformio]
 ;then comment the board you want to ignore
 [platformio]
-;default_envs  = discoveryBoard
-default_envs  = loraBoard
+default_envs  = discoveryBoard
```

Comment:
Heb je ook getest of het compileert voor een lora bord?

Reply:
Als het niet compileert voor LoRa dan kunnen wij het niet pullen dus kan het niet naar develop.